telegram/bot.py

```diff
@@ -5914,6 +5923,140 @@ def get_chat_menu_button(
         )
         return MenuButton.de_json(result, bot=self)  # type: ignore[return-value, arg-type]
 
+    @log
+    def create_invoice_link(
+        self,
+        title: str,
+        description: str,
+        payload: str,
+        provider_token: str,
+        currency: str,
+        prices: List["LabeledPrice"],
+        max_tip_amount: int = None,
+        suggested_tip_amounts: List[int] = None,
+        provider_data: Union[str, object] = None,
+        photo_url: str = None,
+        photo_size: int = None,
+        photo_width: int = None,
+        photo_height: int = None,
+        need_name: bool = None,
+        need_phone_number: bool = None,
+        need_email: bool = None,
+        need_shipping_address: bool = None,
+        send_phone_number_to_provider: bool = None,
+        send_email_to_provider: bool = None,
+        is_flexible: bool = None,
+        timeout: ODVInput[float] = DEFAULT_NONE,
+        api_kwargs: JSONDict = None,
+    ) -> str:
+        """Use this method to create a link for an invoice.
+
+        .. versionadded:: 13.13
+
+        Args:
+            title (:obj:`str`): Product name. :tg-const:`telegram.Invoice.MIN_TITLE_LENGTH`-
```

Comment:
```suggestion
            title (:obj:`str`): Product name. :attr:`telegram.Invoice.MIN_TITLE_LENGTH`-
```
tg-const is not available here, so maybe either use `attr`, or just substitute the number in (hardcode - or should I say _harddoc_)